src/content/1.7/modules/concepts/commands.md

```diff
@@ -9,11 +9,15 @@ weight: 10
 Since version 1.7 of PrestaShop, everyone have access to the PrestaShop console using the following instruction in a terminal:
 
 ``
-./bin/console # or ./app/console for PrestaShop < 1.7.3
+./bin/console
 ``
 
 Since v1.7.5, you can add and provide your own commands into the PrestaShop console using modules.
 
+{{% notice warning %}}
+If you use PrestaShop Core legacy classes, it will not works yet. We will fix it soon.
```

Comment:
The idea is there but it's a bit "scary" when said like this 😛 may I suggest something ?

```suggestion
If you load and use PrestaShop Core legacy classes such as an ObjectModel within a Command context, you might run into issues. This is a known limitation of the Commands.
Removing this limitation is being explored for future PS versions.
```